test/x/track_test.rb

```diff
@@ -4,11 +4,8 @@
 module X
   class TrackTest < Minitest::Test
     def test_tracks
-      f= './test/fixtures/xapi_v3_tracks.json'
```

Comment:
OK, I'll take a look at this and see what's going on.

Thanks for being so patient figuring out all of this. A big part of why I'm doing these rewrites/refactors is because there's a lot of tangled mess at the moment, and just starting the process uncovers a lot of the existing problems.